src/format.rs

```diff
@@ -1,9 +1,7 @@
-use cpp_demangle;
```

Comment:
interesting, i'm surprised this doesn't warn

Reply:
`cargo check` didn't warn about this. `cargo clippy` did.